stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -439,22 +437,22 @@ public struct ${Self}<Pointee>
     Builtin.bindMemory(rawPtr, count._builtinWordValue, Pointee.self)
     return UnsafeMutablePointer(rawPtr)
   }
+  
+  @available(swift, obsoleted: 5.0.0, message: "Swift currently only supports freeing entire heap blocks, use deallocate() instead")
```

Comment:
We should deprecate all these methods that are going away, so users get information about the change even before they've migrated to Swift 5. You can add:
```swift
@available(swift, deprecated, renamed/message: "same as used for the obsoleted attribute") 
```

Reply:
Well, the compiler doesn't actually like that, but you can do even better. Add a deprecated parameter alongside the obsoleted one:
```swift
@available(swift, deprecated: 4.1, obsoleted: 5.0.0, message: "Swift currently only supports freeing entire heap blocks, use deallocate() instead")
 ```

Reply:
did not know you could do that